extensions/blocks/opentable/icon.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/components';
+
+export default (
+	<SVG height="18" viewBox="0 0 25 18" width="25" xmlns="http://www.w3.org/2000/svg">
+		<Path
+			d="m123.5 77.2858191c0-1.2028985.974684-2.1783159 2.176848-2.1783159 1.202165 0 2.177215.9754174 2.177215 2.1783159 0 1.2032655-.97505 2.1786828-2.177215 2.1786828-1.202164 0-2.176848-.9754173-2.176848-2.1786828zm15.29169 2.1786828c-1.202349 0-2.176849-.9754173-2.176849-2.1786828 0-1.2028985.9745-2.1783159 2.176849-2.1783159 1.202164 0 2.177215.9754174 2.177215 2.1783159 0 1.2032655-.975051 2.1786828-2.177215 2.1786828zm.000183-10.8924968c-4.809209 0-8.708127 3.901486-8.708127 8.713814 0 4.8128784 3.898918 8.7141809 8.708127 8.7141809 4.809576 0 8.708127-3.9013025 8.708127-8.7141809 0-4.812328-3.898551-8.713814-8.708127-8.713814z"
+			fill="#c93642"
```

Comment:
@lessbloat what do you think?

Reply:
This has come up a couple of times. Many of the core embed blocks are colored, which is why we went colored with this block. Here's [a monochrome version](https://p111.p2.n0.cdn.getcloudapp.com/items/llu4gY5b/OpenTable-logo-mono.svg). Happy to swap it over.

Reply:
I don't think that's a big issue, but since all other Jetpack block icons are monochrome at the moment, I think it may make sense to keep following that rule so no icon stands out when looking at all blocks in the Jetpack category?